dlrs/clearlinux/tensorflow_2/mkl/scripts/install_tensorflow_2.0.sh

```diff
@@ -55,58 +53,25 @@ run() {
 python_pkgs(){
 # install dependencies for tensorflow build
   pip install pip six wheel numpy setuptools mock "future>=0.17.1"
-  pip install keras_applications==1.0.6 --no-deps
-  pip install keras_preprocessing==1.0.5 --no-deps
+  pip install keras_applications==1.0.8 --no-deps
+  pip install keras_preprocessing==1.1.0 --no-deps
 }
 
 get_project() {
-  git clone https://github.com/${PROJECT}/${PROJECT}.git 
+  git clone --depth 1 -b ${TF_TAG} https://github.com/${PROJECT}/${PROJECT}.git 
   cd tensorflow && git checkout ${TF_TAG}
 }
 
 apply_patch() {
 	# apply patches/fixes to TF
-	# patch to upgrade curl
-	PATCHES=( 33064 )
-	GIT_URL="https://patch-diff.githubusercontent.com/raw/tensorflow/tensorflow/pull/"
-	# git default config
+
+  # git default config
 	git config --global user.email "example@example.com"
 	git config --global user.name "example@example.com"
-	for ptch in "${PATCHES[@]}"; do
-			wget $GIT_URL$ptch.patch && git am $ptch.patch
-	done
-  # fix for: https://github.com/tensorflow/tensorflow/issues/32026
-  echo "Adding fix for: https://github.com/tensorflow/tensorflow/issues/32026"
-  sed -i 's/RUY_DONOTUSEDIRECTLY_AVX512 1/RUY_DONOTUSEDIRECTLY_AVX512 0/g' /tensorflow/tensorflow/lite/experimental/ruy/platform.h
-  
-  # fix for https://github.com/clearlinux/distribution/issues/1151
-  wget https://raw.githubusercontent.com/clearlinux-pkgs/tensorflow/master/0001-Add-grpc-fix-for-gettid.patch 
-  git am 0001-Add-grpc-fix-for-gettid.patch
- 
 }
 
 build () {
-  # configure and build tensorflow avx2 and avx512 versions
-  # build avx2 TF
-  mkdir -p /tmp/tf/avx2
-  export ARCH=skylake
-  export TUNE=skylake
-  export TF_BUILD_MAVX=MAVX2
-  export CC_OPT_FLAGS="-march=${ARCH} -mtune=${TUNE}"
-  export CFLAGS="$CFLAGS -march=${ARCH}"
-  export CXXFLAGS="$CXXFLAGS -march=${ARCH}"
```

Comment:
Here as well, if we can comment the build for avx2 rather than remove it, it would be good if anyone have to build avx2 , they can just use this sections, in the future, we can add a toggle for it.

Reply:
None of the avx2 code was tested as part of v6 release. I think it would be better to leave this as a note in a separate file or readme rather than part of source

Reply:
same as my above comment, only the env vars for historical purposes and also for anyone who wants to build avx2 images, this can be used as a starting point.

Reply:
Let's have it here itself, and mention that the documentation comment above the line 91 on this possibility, its better to have the documentation as part of the code, we have internal builds of avx2 so that should be fine.

Reply:
add in as comments below avx512 install